kubetest/gke.go

```diff
@@ -66,7 +65,9 @@ type gkeNodePool struct {
 
 type gkeDeployer struct {
 	project         string
-	location        string
+	zone            string
+	region          string
+	locationArg     string
```

Comment:
Why do you name it locationArg? Why not leave location?

Reply:
Changed it. Thought that was more apt as it's not the value of the location exactly, but the argument (i.e --zone=foobar)